app/channel.go

```diff
@@ -340,6 +340,30 @@ func (a *App) createGroupChannel(userIds []string, creatorId string) (*model.Cha
 	}
 }
 
+func (a *App) GetGroupChannel(userIds []string) (*model.Channel, *model.AppError) {
+	if len(userIds) > model.CHANNEL_GROUP_MAX_USERS || len(userIds) < model.CHANNEL_GROUP_MIN_USERS {
+		return nil, model.NewAppError("GetGroupChannel", "api.channel.create_group.bad_size.app_error", nil, "", http.StatusBadRequest)
+	}
+
+	var users []*model.User
+	if result := <-a.Srv.Store.User().GetProfileByIds(userIds, true); result.Err != nil {
+		return nil, result.Err
+	} else {
+		users = result.Data.([]*model.User)
+	}
+
+	if len(users) != len(userIds) {
```

Comment:
And this check?

Reply:
Something similar here, if you can't create the group channel with a not existing users, a group channel must have all the users. The exception would be the permanent deletion of a user. With this check If any user is permanently deleted from the database, it won't allow you to access to the group channel. Without this check, you will be able to see it (at least, through the API).

Reply:
OK, so I take your point, but I don't want to change the existing behaviour in this PR, so I'm going to leave this as-is to be consistent with the `CreateGroupChannel` function that it is being used alongside (and is copy-pasted from). If we want to fix the issue you've highlighted, it should be a separate ticket.

Reply:
Fair enough